File: TugasPraktikum03/Bahasa C/PRAK305-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int main() {
    int detik;
    printf("Input: \n");
    scanf("%d", &detik);

    int hari = detik / 86400;
    int hari_sisa = detik % 86400;
    int jam = hari_sisa / 3600;
    int detik_sisa = hari_sisa % 3600;
    int menit = detik_sisa / 60;
    detik = detik_sisa % 60;

    if (hari >= 1) {
        printf("Output:\n");
        printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);
    } else {
        printf("Output:\n");
        printf("%02d:%02d:%02d\n", jam, menit, detik);
    }

    return 0;
}

File: TugasPraktikum03/Bahasa Python/PRAK305-2310817320013-AikoAnatashaWendiono.py
detik = int(input("Input: \n"))

hari = detik // 86400
hari_sisa = detik % 86400
jam = hari_sisa // 3600
detik_sisa = detik % 3600
menit = detik_sisa // 60
detik = detik_sisa % 60

if (hari >= 1):
    waktu_format = f"{hari:01d} hari {jam:02d}:{menit:02d}:{detik:02d}"
    print("Output:")
    print(waktu_format)
else:
    waktu_format = f"{jam:02d}:{menit:02d}:{detik:02d}"
    print("Output:")
    print(waktu_format)
    